<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Программа для прогнозирования рыночных цен</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Загрузки/chart02.ico</normaloff>../../Загрузки/chart02.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>551</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="accessibleName">
      <string comment="График"/>
     </property>
     <attribute name="title">
      <string>График</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>521</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Таблица</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>441</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Создать нейронную сеть</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>280</y>
        <width>85</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonAddRow">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonDeleteRow">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>80</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonPasteRow">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>50</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Вставить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonUploadData">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>110</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Загрузить
новые 
данные
из вне</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>95</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
